.travis.yml

```diff
@@ -19,12 +19,34 @@ matrix:
   allow_failures:
     -name: azurepro-16.04
   include:
+    - if: env(UACLIENT_BEHAVE_AWS_SECRET_ACCESS_KEY)
```

Comment:
I think we should split these into two separate jobs, since this check only for the existence of aws keys and not azure. This means that we may have stale azure vms that will not be cleaned because we are not providing aws keys 9Even though this scenario is not the one we have right now, I think we should be able to support it)